src/frontend/components/TeacherLiveInfoBar/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Heading, Paragraph } from 'grommet';
+import React from 'react';
+import { Nullable } from 'utils/types';
+
+interface TeacherLiveInfoBarProps {
+  title: string;
+  startDate: Nullable<string>;
+}
+
+export const TeacherLiveInfoBar = ({
+  title,
+  startDate,
+}: TeacherLiveInfoBarProps) => {
+  return (
+    <Box direction="column" flex>
+      <Heading
+        color="blue-active"
+        level="2"
+        margin={{ bottom: 'small' }}
+        size="small"
+        truncate
+      >
+        {title}
+      </Heading>
+
+      <Box direction="row">
+        {startDate && (
+          <Paragraph
+            color="blue-active"
+            margin={{ right: 'large', bottom: 'none' }}
+            size="small"
+          >
+            {/* video.started_at */}
+            {startDate}
+          </Paragraph>
+        )}
+      </Box>
```

Comment:
Huum,I don't know.
This is quite legacy since this is done this way for the student.

This was done this way because we where suppose to display more than just the schedule date like the current length of the live etc but those informations are not available yet.